servers/quarkus-server/src/test/java/org/projectnessie/server/TestMetricsTags.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.server;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.projectnessie.quarkus.tests.profiles.QuarkusTestProfilePersistInmemory;
+
+@QuarkusTest
+@TestProfile(TestMetricsTags.Profile.class)
+class TestMetricsTags extends AbstractQuarkusRestWithMetrics {
+
+  @Test
+  void smokeTestMetricsWithTags() {
```

Comment:
There is something strange happening with V1: the version store metrics are absent. Not sure how this is related to my change to labels, I need to investigate.

Reply:
Hmm some tests seem sensitive to test order. You need to exercise the version store first before running them.

Reply:
Yea - the metrics tests are ticky